bigbone/src/main/kotlin/social/bigbone/api/entity/MediaAttachment.kt

```diff
@@ -73,8 +81,111 @@ data class MediaAttachment(
          * Contains the coordinates to be used for smart thumbnail cropping.
          */
         @SerialName("focus")
-        val focus: Focus? = null
-    )
+        val focus: Focus? = null,
+
+        /**
+         * Duration of this media attachment as a String.
+         * Only available for audio and video.
+         */
+        @SerialName("length")
+        val length: String? = null,
+
+        /**
+         * Duration of this media attachment as a Float.
+         * Only available for audio and video.
+         */
+        @SerialName("duration")
+        val duration: Float? = null,
+
+        /**
+         * Frame rate of this media attachment in frames per second.
+         * Only available for video or moving pictures.
+         */
+        @SerialName("fps")
+        val frameRate: Int? = null,
+
+        /**
+         * Size of this media attachment.
+         * Readable string version, like 640x480.
+         */
+        @SerialName("size")
+        val size: String? = null,
+
+        /**
+         * Width of this media attachment.
+         */
+        @SerialName("width")
+        val width: Int? = null,
+
+        /**
+         * Height of this media attachment.
+         */
+        @SerialName("height")
+        val height: Int? = null,
+
+        /**
+         * Aspect ratio of this media attachment.
+         */
+        @SerialName("aspect")
+        val aspectRatio: Int? = null,
```

Comment:
This aspect ratio should also be of type `Double`.

Reply:
✅